src/posts/blogs/2019/2019-12-04-layer5-landscape-spectrums-revisited/index.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: "Layer5: Landscape Spectrums Revisited"
+subtitle: ""
+date: 2019-12-04 18:10:00 +0000
+author: Rafi Ungar
+thumbnail: ./landscape.png
+tags: Community, Projects, News
+published: true
```

Comment:
Is there a description that we can include for SEO?

Reply:
A larger issue we can address later. Something that is good to give consideration to as we go, though.